test/studies/bale/indexgather/ig-auto-agg.ml-execopts

```diff
@@ -9,9 +9,6 @@ comm_sub = os.getenv('CHPL_COMM_SUBSTRATE')
 ugni = comm == 'ugni'
 gn_aries = comm == 'gasnet' and comm_sub  == 'aries'
```

Comment:
Can remove all the config checking and probably comment above if you want too.

Reply:
Yep! I just switched to a plain-old-execopts file